client/components/steps/business-address.jsx

```diff
@@ -66,6 +82,11 @@ module.exports = React.createClass( {
 						<input className="welcome__business-address--input" type="text" name="business_city" id="business-city" value={ this.state.business_city } onChange={ this.handleChange } placeholder="City" required/>
 						<input className="welcome__business-address--input" type="text" name="business_state" id="business-state" value={ this.state.business_state } onChange={ this.handleChange } placeholder="State" />
 						<input className="welcome__business-address--input" type="text" name="business_zip" id="business-zip" value={ this.state.business_zip } onChange={ this.handleChange } placeholder="Zip" required />
+						<div className="welcome__business-address--woo-container">
+							<input className="welcome__business-address--checkbox" type="checkbox" name="sell_online" id="sell-online" checked={ this.state.sell_online } onChange={ this.handleChange } />
```

Comment:
`id="sell-online"` should probably be changed to `id="sell_online"` , as it currently does not match the `for` attribute of the corresponding label.